### PR TITLE
SW-2794 Added Organization*Services

### PR DESCRIPTION
**Pull request by karthikbtf**
- OrganizationService for organization CRUD
- OrganizationUserService for organization user CRUD
- renamed AllOrganizationRoles as OrganizationRole
- updated/tested all changes to calls

### CODE REVIEW

File: src/services/index.ts [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
 import PreferencesService from './PreferencesService';
 import UserService from './UserService';
 
-export { CachedUserService, HttpService, PreferencesService, UserService };
+export type { Response } from './HttpService';
+
+export {
+  CachedUserService,
+  HttpService,
+  OrganizationUserService,
```
Comment (constanzauanini) [inline src/services/index.ts:13]: nitpick: if these are in alphabetical order, OrganizationService should be before OrganizationUserService

**Review by constanzauanini — APPROVED** [review]
Looks good to me!

File: src/services/index.ts (hunk 1 above)
Comment (karthikbtf) [reply src/services/index.ts:13]: thanks, that's how i would usually sort..  in the strings file we are sorting differently.. applied that here. will fix here.

Comment (karthikbtf) [reply src/services/index.ts]:
```
node -e 'console.log(["OrganizationService", "OrganizationUserService"].sort())'
[ 'OrganizationService', 'OrganizationUserService' ]
```
oh well...

**Pull request merged**